composer.lock

```diff
@@ -3948,16 +3957,16 @@
         },
         {
             "name": "prestashop/ps_themecusto",
-            "version": "v1.0.9",
+            "version": "v1.0.10",
```

Comment:
Is this expected?

Reply:
If there was an update I don't think it's a problem to integrate it in the zip